src/system/SystemConfig.h

```diff
@@ -40,6 +40,33 @@
 /* Platform include headers */
 #include <BuildConfig.h>
 
+/* Include a project-specific configuration file, if defined.
```

Comment:
Maybe we should differentiate this comment from the similar one below.  E.g. :
  
_Include a project-specific **Weave** configuration file file, if defined._

Reply:
done